nwb_conversion_tools/datainterfaces/ecephys/neuroscope/neuroscopedatainterface.py

```diff
@@ -1,8 +1,10 @@
 """Authors: Cody Baker and Ben Dichter."""
 from pathlib import Path
-from typing import Optional
+from typing import Optional, Union
 
 import spikeextractors as se
+import spikeinterface as si
```

Comment:
```suggestion
from spikeinterface.core.old_api_utils import OldToNewRecording
```
One trick that can be done to return horizontal line usage across primary code blocks below imports is to use relative imports whenever there aren't so many that they don't span multiple lines in `black` (so like, 4 specific function imports).